syz-ci/syz-ci.go

```diff
@@ -265,6 +271,34 @@ func main() {
 	}
 }
 
+func deprecateAssets(cfg *Config, shutdownPending chan struct{}) {
+	dash, err := dashapi.New(cfg.DashboardClient, cfg.DashboardAddr, cfg.DashboardKey)
+	if err != nil {
+		log.Logf(0, "deprecateAssets: failed to create dashapi: %v", err)
+		return
+	}
+	storage, err := asset.StorageFromConfig(cfg.AssetStorage, dash)
+	if err != nil {
+		log.Logf(0, "deprecateAssets: failed to create asset storage: %v", err)
```

Comment:
Nobody is reading all logs every day, so just logging an error is not much different from silently ignoring it. Why not log.Fatalf?